src/mono/mono/mini/interp/interp.c

```diff
@@ -4164,6 +4164,9 @@ mono_interp_exec_method (InterpFrame *frame, ThreadContext *context, FrameClause
 			}
 			cmethod = del_imethod;
 			if (!is_multicast) {
```

Comment:
does multicast not need this deathgrip var because multicast invokes are done by a helper method that has 'this' on the stack?